tests/integration/targets/rds_instance/inventory

```diff
@@ -1,12 +1,18 @@
+# inventory names shortened down to fit resource name length limits
 [tests]
+# processor feature tests
+processor
+# restore instance tests
+restore
+# security groups db tests
+sgroups
+# modify complex tests
+complex
+# other tests
 states
 modify
-modify_complex
-processor_features
-read_replica
-vpc_security_groups
-restore_instance
 tagging
+replica
```

Comment:
renamed these to ensure we sit in the resource length limits as there was a load of issues on the `3.0.0` release around these limits being hit leading to these tests being disabled.